go-controller/pkg/ovn/master.go

```diff
@@ -295,21 +295,11 @@ func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 		loadBalancerGroup := nbdb.LoadBalancerGroup{
 			Name: types.ClusterLBGroupName,
 		}
-		loadBalancerGroupRes := []nbdb.LoadBalancerGroup{}
+		// Create loadBalancerGroup is there is not one. Since this table is indexed by name, there is no need to
+		// mention that field in OnModelUpdates, nor use a ModelPredicate.
 		opModels := []libovsdbops.OperationModel{
 			{
-				Model:          &loadBalancerGroup,
-				ModelPredicate: func(lbg *nbdb.LoadBalancerGroup) bool { return lbg.Name == types.ClusterLBGroupName },
-				OnModelUpdates: []interface{}{
-					&loadBalancerGroup.Name,
-				},
-				ExistingResult: &loadBalancerGroupRes,
-				DoAfter: func() {
-					if len(loadBalancerGroupRes) > 0 {
-						loadBalancerGroup.UUID = loadBalancerGroupRes[0].UUID
-					}
-				},
-				ErrNotFound: false,
+				Model: &loadBalancerGroup,
```

Comment:
@dceara less is more! ;)

Reply:
Nice!  I tried your PR out and it seems to work as expected, at least from LB Group perspective.  I didn't review the internals though.